src/main/java/uk/gov/hmcts/reform/civil/utils/CaseStateUtils.java

```diff
@@ -11,12 +12,15 @@ private CaseStateUtils() {
         //NO-OP
     }
 
-    public static boolean shouldMoveToInMediationState(CaseData caseData, boolean carmEnabled) {
-        if (carmEnabled && SpecJourneyConstantLRSpec.SMALL_CLAIM.equals(caseData.getResponseClaimTrack())) {
+    public static boolean shouldMoveToInMediationState(CaseData caseData, FeatureToggleService featureToggleService) {
+        if (caseData.isFullAdmitClaimSpec()) {
+            return false;
+        }
+        if (featureToggleService.isCarmEnabledForCase(caseData) && SpecJourneyConstantLRSpec.SMALL_CLAIM.equals(caseData.getResponseClaimTrack())) {
```

Comment:
Since you are working on this it might be worth it checking whether you can remove the feature flag